Bio/Align/Applications/_Guidance2.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 by Rob Gilmore and Shaurita Hutchins. All rights reserved.
+# Based on ClustalOmega wrapper copyright 2011 by Andreas Wilm.
+#
+# Wrapper for Guidance2 by Rob Gilmore (2017).  http://guidance.tau.ac.il/ver2/
+# Used _ClustalOmega.py as template.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+
+"""Command line wrapper for the multiple alignment program, GUIDANCE2.
+It weights, filters or masks unreliably aligned positions in multiple sequence alignments.
+"""
+
+from __future__ import print_function
+
+import os
+from Bio.Application import _Option, AbstractCommandline
+
+
+class Guidance2Commandline(AbstractCommandline):
+    u"""Command line wrapper for GUIDANCE2.
```

Comment:
Is there really any unicode in this docstring? If it is only funny dash characters in the reference page numbers, they can be replaced with minus signs.